ArchUnitNET/Fluent/Syntax/Elements/Members/FieldMembers/FieldMembersShouldConjunctionWithoutBecause.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ArchUnitNET.Domain;
+
+namespace ArchUnitNET.Fluent.Syntax.Elements.Members.FieldMembers
+{
+    public class
+        FieldMembersShouldConjunctionWithoutBecause : ObjectsShouldConjunctionWithoutBecause<FieldMembersShould,
```

Comment:
you can make code more readable if you put class name on a previous line and move parent class to the next line

Reply:
changed in new version